functions/src/users.ts

```diff
@@ -0,0 +1,7 @@
+import * as functions from "firebase-functions";
+import { userCollection } from "./firestore-collections";
+
+export const user = functions.auth.user().onDelete((user) => {
```

Comment:
lets call this something that makes it more clear what it is doing like `onDeleteUser` or something like that

Reply:
got it

Reply:
Why was this marked as resolved?

Reply:
Sorry, I made changes but forgot to push them